tools/GenerateHelp.ps1

```diff
@@ -11,21 +11,25 @@ Param(
 
 Import-Module "$PSScriptRoot\HelpGeneration\HelpGeneration.psm1"
 
-$HelpFolders = Get-ChildItem "help" -Recurse -Directory | where { $_.FullName -like "*Debug*" }
+$HelpFolders = Get-ChildItem "help" -Recurse -Directory | where { $_.FullName -like "*Debug*" -or $_.FullName -like "*Release*" }
 
 # ---------------------------------------------------------------------------------------------
 
 if ($ValidateMarkdownHelp)
 {
     New-Item -Path "$PSScriptRoot\HelpGeneration\Exceptions" -Name NewValidateHelpExceptions.csv -ItemType File -Force | Out-Null
 
-    Add-Content "$PSscriptRoot\HelpGeneration\Exceptions\NewValidateHelpExceptions.csv" "Module,Target,Description"
+    Add-Content "$PSScriptRoot\HelpGeneration\Exceptions\NewValidateHelpExceptions.csv" "Module,Target,Description"
```

Comment:
Generally, Exceptions are for expected errors, I think the error csvs are being copied to the build directory, and we should mnake sue of thsi same location  Also, do we have any mechanism for suppressions?